### PR TITLE
Package fdkaac.0.3.2

### PR DESCRIPTION
**Pull request by toots**
### `fdkaac.0.3.2`
Fraunhofer FDK AAC Codec Library
The FDK AAC Codec Library For Android contains an encoder implementation of the Advanced Audio Coding (AAC) audio codec.

---
* Homepage: https://github.com/savonet/ocaml-fdkaac
* Source repo: git+https://github.com/savonet/ocaml-fdkaac.git
* Bug tracker: https://github.com/savonet/ocaml-fdkaac/issues

---
:camel: Pull-request generated by opam-publish v2.1.0

### CODE REVIEW

File: packages/conf-fdkaac/conf-fdkaac.1/opam [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  ["libfdk-aac-dev"] {os-family = "debian" | os-family = "ubuntu"}
+  ["fdk-aac-dev"] {os-family = "alpine"}
+  ["fdk-aac-free-devel"] {os-distribution="centos" | os-family = "fedora" | os-family = "suse"}
+  ["fdk-aac"] {os-distribution = "nixos" | os = "freebsd" | os = "macos" & os-distribution = "homebrew"}
+  ["libfdk-aac"] {os-distribution = "arch"}
+]
+synopsis: "Virtual package relying on fdk-aac"
+description:
+  "This package can only install if the fdk-aac is installed on the system."
+flags: conf
```
Comment (raphael-proust) [inline packages/conf-fdkaac/conf-fdkaac.1/opam:21]:
```suggestion
flags: conf
x-ci-accept-failures: ["oraclelinux-8"]
```

The package doesn't seem to be found by the CI on oracle8

**Review by raphael-proust — COMMENTED** [review]
There are also some errors on some tests which are related to unavailability on debian I think:

```
# The following system packages are needed:
libfdk-aac-dev
pkg-config
The following command needs to be run through "sudo":
    apt-get install -qq -yy libfdk-aac-dev pkg-config
E: Unable to locate package libfdk-aac-dev
OS package installation failed
```

This seems to be more of an issue with the CI: it detects correctly that the package is unavailable for some jobs but not for

- opam-2.0-ocaml-4.14
- opam-2.0-ocaml-5.0

-----------------------------------------

The macos CI failures seem to be due to the CI runner: git lock file.

File: packages/conf-fdkaac/conf-fdkaac.1/opam (hunk 1 above)
Comment (masternet0) [reply packages/conf-fdkaac/conf-fdkaac.1/opam:21]: abour depian need this deb-multimedia-keyring to install libfdk-aac-dev

echo "deb http://www.deb-multimedia.org bullseye main" >> /etc/apt/sources.list
apt-get -y update -oAcquire::AllowInsecureRepositories=true
apt-get -y install deb-multimedia-keyring

test on Debian 11

Comment (toots) [reply packages/conf-fdkaac/conf-fdkaac.1/opam:21]: Yes, the `fdk-aac` packages are not always available b/c of the restrictive license. It's actually pretty new that they are available on rpm. 

The packages are also available on debian non-free but it's not enabled by default: https://packages.debian.org/sid/libfdk-aac-dev

File: packages/fdkaac/fdkaac.0.3.2/opam [hunk 2]
```diff
@@ -20,25 +23,18 @@ build: [
...
-  ["libfdk-aac-dev"] {os-family = "debian"}
-  ["fdk-aac"] {os = "macos" & os-distribution = "homebrew"}
-]
 url {
-  src: "https://github.com/savonet/ocaml-fdkaac/archive/0.3.2.tar.gz"
+  src: "https://github.com/savonet/ocaml-fdkaac/archive/v0.3.3.tar.gz"
   checksum: [
-    "md5=34800a5a6f7e055699df5d8244679e40"
-    "sha512=fd66e54587b9d3162408ea923a45ca444885668ae56cd42a8ead378ed4579d86edd24e71ff0f6cf2bffb5a14d4cafb4b6c1a9577b16d13c3a8d47fa531cd1144"
+    "md5=c49d8e20590ff0d6c264534c9b26a875"
```
Comment (mseri) [inline packages/fdkaac/fdkaac.0.3.2/opam:37]: Why are you editing release 0.3.2 instead of releasing a new one?

Comment (mseri) [reply packages/fdkaac/fdkaac.0.3.2/opam:37]: I think this is meant to be 0.3.3

**Comment by toots** [comment]
Whoops thanks for catching this @mseri. New PR is here: https://github.com/ocaml/opam-repository/pull/22862

**Pull request closed without merging**